finetuner/tuner/onnx.py

```diff
@@ -0,0 +1,158 @@
+import numpy as np
+import onnxruntime
+
+from typing import (
+    Any,
+    List,
+    Optional,
+    Tuple,
+    Union,
+)
+
+from finetuner.helper import get_framework
+
+
+def to_onnx(
+    embed_model: 'AnyDNN',
+    path: str,
+    input_shape: Union[Tuple[int], List[int]],
+    batch_size: Optional[Union[int, None]] = None,
```

Comment:
I think `batch_size` should not be an argument here. It's an extra parameter for the user to consider and it doesn't offer anything